recipes/gotranx/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "gotranx" %}
+{% set name_underscore = "gotranx" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name_underscore }}-{{ version }}.tar.gz"
+  sha256: a00f6a94a6038de21ceee525e8e1870077a4473a4f5e272f6f8696bce4bb28cd
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - "gotranx=gotranx.cli:app"
+  script: python -m pip install -v --no-deps .
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - attrs
+    - lark
+    - pint
+    - rich-click
+    - structlog
+    - sympy
+    - typer
+    - toml
+test:
+  requires:
+    - python {{ python_min }}
+
+  imports:
+    - gotranx
+
+  source_files:
+    - tests/
+    - pyproject.toml
+
+  commands:
+    - gotranx --help
```

```suggestion
    - gotranx --help
    - pip check
```

and add `pip` under `test/requires`